.travis.yml

```diff
@@ -44,30 +44,27 @@ matrix:
      language: python
      python: 3.5
      env: TESTENV=build_console
-     before_install: make deps
+     before_install:
+       - make deps
+       - pip install -r requirements_dev.txt
      script: make build_console
 
    - os: osx
      osx_image: Xcode 8.3.3
-     env: TESTENV=mac TOXENV=py27,py37
      language: generic
      before_install:
-        - export PATH="/usr/local/opt/python@2/libexec/bin:$PATH"
+        - export PATH="/Users/travis/miniconda3/bin:$PATH"
         - make deps
-     before_script:
-        - export PATH="/usr/local/bin/:./:$PATH"
-        - export DISPLAY=:0.0
-        - Xvfb ${DISPLAY} -screen 0 800x600x16 & PID=$!
```

Comment:
doesn't this break running the gui smoketest on mac?

Reply:
As far as I know, Xvfb is a X11 windows display manager (and we're not using X11 on Mac), should be completely unnecessary for the Mac build and the smoke tests seem to run fine without it.

Reply:
ok, probably there is some "display" available in travis' mac environment by default, then. Smoke test running without any special errors/warnings is enough.